File: springboot2.x-learning/springboot-data-jpa-multi/src/main/java/com/rtxtitanv/repository/secondary/SecondaryUserRepository.java
package com.rtxtitanv.repository.secondary;

import com.rtxtitanv.model.secondary.SecondaryUser;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.jpa.repository.JpaSpecificationExecutor;

/**
 * @author rtxtitanv
 * @version v1.0.0
 * @name com.rtxtitanv.repository.secondary.SecondaryUserRepository
 * @description SecondaryUserRepository用于操作从库用户表
 * @date 2020/1/7 19:37
 */
public interface SecondaryUserRepository extends JpaRepository<SecondaryUser, Long>, JpaSpecificationExecutor<SecondaryUser> {
}

File: springboot2.x-learning/springboot-data-jpa-multi-kotlin/src/main/kotlin/com/rtxtitanv/repository/secondary/SecondaryUserRepository.kt
package com.rtxtitanv.repository.secondary

import com.rtxtitanv.model.secondary.SecondaryUser
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.jpa.repository.JpaSpecificationExecutor

/**
 * @name com.rtxtitanv.repository.secondary.SecondaryUserRepository
 * @description SecondaryUserRepository用于操作从库用户表
 * @author rtxtitanv
 * @date 2020/2/5 19:02
 * @version v1.0.0
 */
interface SecondaryUserRepository : JpaRepository<SecondaryUser, Long>, JpaSpecificationExecutor<SecondaryUser>